ds_exploration/data_download.py

```diff
@@ -0,0 +1,88 @@
+import shutil
+import rasterio
+import fnmatch
+import os
+from pathlib import Path
+from glob import glob
+import numpy as np
+import pandas as pd
+from ds_exploration.plotting_utils import plot_image
+
+os.chdir('../sentinelsat/')
```

Comment:
Do you need to have sentinelsat pulled next to the repo here? Cant we just pip install it?

Reply:
For some reason, `pip install`ing it only gets a part of the package, but not the `products` module.

So I resorted to cloning and installing locally with `[dev]` flag, in order to try with he `nodefilter` option. Still doesn't work yet, though...

Reply:
I mean, with the local clone the code is available and runs, but it still gives an error. I guess they have some problems with the package?

Reply:
Ah I see, the pypi package is updated 2020 june and the github master branch a few days ago for the products file. Could you add the git repo link in the requirements file then instead? So that pip installs from the github repo and gets the latest files containing the products module?

Reply:
ok cool I will do that then